src/cli/cli.rs

```diff
@@ -306,6 +306,24 @@ pub fn build_cli() -> App<'static, 'static> {
                 )
                 .arg(color_arg.clone()),
         )
+        .subcommand(
+            SubCommand::with_name("colorblind")
+                .about("Simulate a color under a certain colorblindness profile")
+                .long_about(
+                    "Convert the given color to how it would look to a person with protanopia, \
+                    deuteranopia, or tritanopia \n\n\
+                     Example:\n  \
+                       pastel random | pastel colorblind deut")
+                .arg(
+                    Arg::with_name("type")
+                        .help("The type of colorblindness that should be simulated (protanopia, \
+                               deuteranopia, tritanopia)")
+                        .possible_values(&["pro", "deut", "tri"])
```

Comment:
Are those common abbreviations for the three types of color blindness or should we maybe just spell them out?

Reply:
I gave this some thought: there is no common abbreviation of these words afaik, but they are admittedly hard to spell. So I made sure to include the full spelling in two places and use reasonable-enough abbreviations in the CLI. Does this sound alright?

Reply:
Yes, that sounds reasonable. Thanks.